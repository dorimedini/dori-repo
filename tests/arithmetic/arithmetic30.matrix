{
	matrix m(2,3) = [1, 2, 3; 4, 5, 6];

	matrix mul_trans(2,2)=m * transpose(m);
	print mul_trans;

	matrix neg(2,3)=-m;
	print neg;
	
}