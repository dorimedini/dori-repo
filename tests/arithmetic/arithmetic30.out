[14,32;32,77]
[-1,-2,-3;-4,-5,-6]
